### ISSUE TITLE
Missing field "organization" in connector config

### ISSUE THREAD

**Issue by panasenco** [fivetran/terraform-provider-fivetran#94]
When trying to use Terraform to set up the new [Gladly](https://fivetran.com/docs/rest-api/connectors/config#gladly) preview connector, I get the following error. The Fivetran API for Gladly requires the "organization" config, but that doesn't seem supported in Terraform yet.

```
╷
│ Error: Unsupported argument
│ 
│   on fivetran_gladly.tf line 28, in resource "fivetran_connector" "gladly":
│   28:     organization = var.gladly_organization
│ 
│ An argument named "organization" is not expected here. Did you mean
│ "organizations"?
╵
```

Please add the field "organization" to the [fivetran_connector config nested schema](https://registry.terraform.io/providers/fivetran/fivetran/latest/docs/resources/connector#nestedblock--config).

Here's my Terraform code:

```terraform
variable "gladly_username" {
  description = "To be passed in during CI/CD as the username to connect to Gladly."
  type        = string
}

variable "gladly_api_token" {
  description = "To be passed in during CI/CD as the API token to connect to Gladly."
  type        = string
}

variable "gladly_organization" {
  description = "To be passed in during CI/CD as the organization to connect to Gladly."
  type        = string
}

resource "fivetran_connector" "gladly" {
  group_id          = fivetran_group.group.id
  service           = "gladly"
  sync_frequency    = 60 # minutes
  paused            = false
  pause_after_trial = true  # <-- SET TO FALSE BEFORE MERGING TO MAIN
  destination_schema {
    name = "gladly"
  }
  config {
    username = var.gladly_username
    api_token = var.gladly_api_token
    organization = var.gladly_organization
  }
  depends_on = [
    fivetran_destination.snowflake,
  ]
}
```

**Comment by beevital** [comment]
Created an internal ticket for this issue.

**Comment by beevital** [comment]
Update included into the next version v0.6.13, release planned at March 1.
